### DISCUSSION TITLE
HELP!! Blobbing on prime tower

### DISCUSSION THREAD

**Discussion by AdamDrefs** [Enraged-Rabbit-Community/ERCF_v2#157] [Q&A]
Basic Information:
 Printer Model: Heavily modded ender 3 V2 Neo(with ERCF V2)
 MCU / Printerboard: Creality 4.2.2
 Host / SBC: RPI 4B 1GB
 [klippy.log](https://klipper.discourse.group/uploads/short-url/1fPul20ct8mH9K0lPVX0USLk6xB.log)
klippy.log (3.1 MB)

Describe your issue:
My tip forming is leaving large blobs on the prime tower, even though it doesn’t do it during testing. I have tip forming in slicer disabled and have force tip forming standalone enabled. Whenever I run the tip forming commands manually, they form a nearly perfect tip without excess extrusion or any other issues. However, when I attempt to run even the simplest of prints, it creates a blob on the tower. I think that for some reason it is not pulling back the filament before tip forming, causing the first cooling move to push out filament, but I have no idea why this would occur(especially just during prints) nor how I would fix it. I have tried ooze reduction, telling it that the filament is lower than it should be, and slowing everything way down in case of an extruder error.

Any knowledge as to why this is occurring or how I can fix it would be greatly appreciated, since these blobs make my prime tower fail keeping me from utilizing my MMU.

Also, whenever it creates the blob, it also fails to fully unload the filament from the extruder, if that is of any help.

**Comment by AdamDrefs** [comment]
Solved it myself! There seems to be an internal issue with the motor sync function causing it to not sync during tip forming, causing the extruders to fight against each other. I will close this thread and open a new one in the happy hare github to bring this up.

**Comment by norris108** [comment]
For reference [here](https://github.com/moggieuk/Happy-Hare/issues/567) is the Happy Hare issue
